src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+import java.util.*;
 public class Duke {
```

Comment:
can consider using more classes - will make things much simpler to work with in the later stages